File: RecommendationEngine/Models/Application/Job.cs
using System;

namespace Models.Application
{
    public class Job
    {
        public int Id { get; set; }
        public int Duration { get; set; }
        public int ConfiguredRecommendationId { get; set; }
        public string ConfiguredRecommendationTitle { get; set; } 
        public string Status { get; set; }
        public DateTime Timestamp { get; set; }
    }
}

File: RecommendationEngine/RecommendationEngine/FrontEnd/src/api/models/Job.ts


export interface ConfiguredRecommendationJob {
    id: number,
    duration: number,
    configuredRecommendationId: number,
    configuredRecommendationTitle: string,
    status: string,
    assetName: string,
    timestamp: Date;
}

export interface JobLog{
    id: number,
    time: Date,
    level: string,
    description: string;
}